target/linux/ramips/image/mt7621.mk

```diff
@@ -239,6 +239,20 @@ define Device/edimax_rg21s
 endef
 TARGET_DEVICES += edimax_rg21s
 
+define Device/edimax_ra21s
+  MTK_SOC := mt7621
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := Edimax
+  DEVICE_MODEL := Gemini AC2600 RA21S
```

Comment:
Based on the node name I would expect only "RA21S" here.
Can you comment on the background of this long name, please.

Reply:
The box has a label `Gemini RA21S`. However sticker on the botton has string `Model No.: RA21S`. So I ould change to `RA21S`

Reply:
If I just google it, I find "Edimax RA21S" at standard resellers.
So yes, rename it with "DEVICE_MODEL := RA21S".

If you think that an alternate name would be helpful/necessary, there would also be a syntax for specifying that with adding
DEVICE_ALT0_VENDOR :=
DEVICE_ALT0_MODEL :=

Reply:
> rename it with "DEVICE_MODEL := RA21S".

done

> If you think that an alternate name would be helpful/necessary, there would also be a syntax for specifying that with adding

I will add `ALT0_*` info. I see combination of `Gemini RA21S` in the user manual and it also occurs on the product information page.